File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.py
def solution(strArr):
    # method1
    r=[]
    for i in range(len(strArr)):
        r.append(strArr[i].lower() if i%2==0 else strArr[i].upper())
    # method2
    # answer=[strArr[i].lower() if i%2==0 else strArr[i].upper() for i in range(len(strArr))]
    return r

File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> strArr) {
    vector<string> r;
    string t;
    for(int i=0; i<strArr.size(); i++) {
        if(i%2==0)
            for(int j=0;j<strArr[i].length();j++) {
                t+=tolower(strArr[i][j]);
            }
        else
            for(int j=0;j<strArr[i].length();j++) {
                t+=toupper(strArr[i][j]);
            }
        r.push_back(t);
        t="";
    }
    return r;
}